# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hpijs/hpijs-gcc4-compat.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- hpijs-1.7.1/printerproxy.h.orig	2005-09-08 16:54:55.000000000 -0700
+++ hpijs-1.7.1/printerproxy.h	2005-09-08 16:53:22.000000000 -0700
@@ -47,6 +47,7 @@
 };
 
 
+class Printer;
 
 //PrinterProxy
 //!Provide act on behalf of the printer class for matching, names, voting, etc
